Source/Core/DolphinQt/Config/Mapping/HotkeyControllerProfile.cpp

```diff
@@ -16,10 +16,27 @@ HotkeyControllerProfile::HotkeyControllerProfile(MappingWindow* window) : Mappin
 
 void HotkeyControllerProfile::CreateMainLayout()
 {
-  m_main_layout = new QHBoxLayout();
-
-  m_main_layout->addWidget(CreateGroupBox(
-      tr("Controller Profile"), HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE)));
+  m_main_layout = new QGridLayout;
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %1").arg(1),
+                     HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE_1)),
+      0, 0);
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %2").arg(2),
+                     HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE_2)),
+      0, 1);
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %3").arg(3),
+                     HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE_3)),
+      1, 0);
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %4").arg(4),
```

Comment:
Are you sure this is the right way to format these? I'd assume the %1 is a placeholder for the argument and then the .arg() fills in the argument, so all of these should read "Wii Remote %1". Or just use literal strings and get rid of the .arg().

Reply:
As @jordan-woyak said upper, it must be 1% for the string format.